test/node/common-safe-require.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const tap = require('tap');
+
+const common = require('../../lib/common');
+
+const existingModule = 'tap';
+const nonExistingModule = 'tp';
+
+tap.ok(common.safeRequire(existingModule), 'must require existing module');
+
+tap.equal(common.safeRequire(nonExistingModule), null,
+  'must return \'null\' if module doesn\'t exist');
```

Comment:
It will be better to avoid escaping by using backticks or double quotes here, but this is not mandatory.